### ISSUE TITLE
extract author from novelbin.net

### ISSUE THREAD

**Issue by mapreri** [dteviot/WebToEpub#974]
The novelbin.net plugin doesn't detect the author

However the information is there, see https://novelbin.net/n/mai-kitsune-waifu-nov-1561312583 for example, in 
`/html/body/div/main/div[3]/div[1]/div[1]/div[3]/ul/li[2]/a` which, I think, is selectable by

     document.querySelector("div ul.info-meta").children[1].querySelector("a").text;

Could you possibly add that to the novelbin.net parser? Thank you in advance!

**Comment by dteviot** [comment]
@mapreri 

That doesn't work for https://novelbin.net/n/earths-greatest-magus-nov-298370430.  
Order of items in metadata is different.
So, gets ugly.  Need to look for element with a &lt;h3&gt; tag with contents of "Author:"

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://novelbin.net/n/earths-greatest-magus-nov-298370430
* https://novelbin.net/n/mai-kitsune-waifu-nov-1561312583#tab-chapters-title

For my notes: 36 minutes work

**Comment by mapreri** [comment]
I tried a bunch more of epubs and it works nicely, thank you! :)

**Comment by dteviot** [comment]
@mapreri 

Updated version (0.0.0.154) has been submitted to Firefox and Chrome stores.
Firefox version is available now.
Chrome might be available in 1 to 3 weeks.

**Comment by mapreri** [comment]
Hello,

I wonder, novel-bin.net seems to use a different parser, but after all it is always the same as far as parsing go.  Would it be possible to share this across the two parsers?

**Comment by dteviot** [comment]
@mapreri 
Firstly, please don't add different issues to an existing issue.  It makes keeping track of what has been done versus what still needs to be done difficult. (Especially when an issue has been closed.)  Instead, please create a new issue. Note, adding more details about an existing incident is fine.

That said, to answer your question above.

Do you mean, you think for site novel-bin.net , WebToEpup should use the same parser that is used for novelbin.net?
If so, yes, that can be done.

**Comment by dteviot** [comment]
@mapreri 

Looks like you were right.  The sites have same layout.  I've consolidated the two parsers.

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://novel-bin.net/novel-bin/my-genes-can-evolve-limitlessly-nov2075908803#tab-chapters-title
* https://novelusb.com/novel-book/martial-world-novel-novel

For my notes: 49 minutes work
